### PR TITLE
Cavstool debug slot offset by type

### PR DESCRIPTION
**Pull request by jsarha**
One more commit on top of: https://github.com/zephyrproject-rtos/zephyr/pull/78753

But I made this a separate PR as the review of the PR above is already quite far and I hope to see it merged soon.

### CODE REVIEW

File: soc/intel/intel_adsp/tools/cavstool.py [hunk 1]
```diff
@@ -745,6 +745,9 @@ def debug_offset():
     ( base, stride ) = adsp_mem_window_config()
     return base + stride * 2
 
+def debug_slot_offset(num):
+    return debug_offset() + DEBUG_SLOT_SIZE * (1 + num)
```
Comment (lgirdwood) [inline soc/intel/intel_adsp/tools/cavstool.py:749]: I guess in the future we will need to detect slot size used by FW if it changes, but today using the default is good.

**Review by lgirdwood — DISMISSED** [review]

**Comment by jsarha** [comment]
**### NOTE:**

There is only one relevant commit in this PR, the last. Please do not waste time in reviewing the earlier commits as they have already been approved here: https://github.com/zephyrproject-rtos/zephyr/pull/78753

**Review by dcpleung — DISMISSED** [review]

**Comment by aescolar** [comment]
@jsarha setting dnm label as it would seem this PR needs to go after:
- https://github.com/zephyrproject-rtos/zephyr/pull/78753 

If that is not the case please remove the label

**Comment by jsarha** [comment]
> @jsarha setting dnm label as it would seem this PR needs to go after:
> 
> * [Make cavstool code more shareable #78753](https://github.com/zephyrproject-rtos/zephyr/pull/78753)
> 
> If that is not the case please remove the label

@aescolar , The PR above is now merged and the DNM label should re removed, but it looks like  I do not have the required permission.

**Review by lgirdwood — APPROVED** [review]

**Review by dcpleung — APPROVED** [review]

**Pull request merged**